### PR TITLE
[Pal/Linux-SGX] Remove TODO on fetching SigRLs from IAS in RA-TLS

### PR DESCRIPTION
**Pull request by dimakuv**
## Description of the changes 

SigRLs are Signature Revocation Lists that can be separately fetched from Intel Attestation Service (IAS), to verify that the EPID private key of the user was revoked. Previously, we had a TODO comment to fetch these SigRLs and verify against them. However, this is not needed because the "verify attestation evidence" request to IAS (issued by RA-TLS) already forces IAS to check the EPID key against SigRLs. Thus, fetching SigRLs separately is just an optimization and not needed to be enforced in RA-TLS.

Please check https://api.trustedservices.intel.com/documents/IAS-API-Spec-rev-5.0.pdf, Section 3.2 for the details of "verify attestation evidence" end-point of IAS. There is an example of a revoked EPID group in Section 3.2.3.7. Similarly, the IAS attestation report will return `"isvEnclaveQuoteStatus": "SIGNATURE_REVOKED"` if the EPID signature was revoked, see Section 4.2.1.

Finally, notice that our RA-TLS code errors out on any `isvEnclaveQuoteStatus` that is not `OK` or `GROUP_OUT_OF_DATE`/`CONFIGURATION_NEEDED`/`SW_HARDENING_NEEDED`/`CONFIGURATION_AND_SW_HARDENING_NEEDED`: https://github.com/oscarlab/graphene/blob/d4602cee8c76e4d8fb8002777009e2fc016ba077/Pal/src/host/Linux-SGX/tools/common/attestation.c#L221-L237

In other words, `isvEnclaveQuoteStatus == SIGNATURE_REVOKED` will fail RA-TLS verification. No additional step of fetching SigRLs is thus needed.

## How to test this PR? 

N/A.

---
This change is [Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)

### CODE REVIEW

**Review by mkow — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "TODO" found in commit messages' one-liners (waiting on @dimakuv)

___
*[Pal/src/host/Linux-SGX/tools/ra-tls/ra_tls_verify_epid.c, line 226 at r1](https://reviewable.io/reviews/oscarlab/graphene/2575#-MfFbXCj2ZAXvbPd7nho:-MfFbXCj2ZAXvbPd7nhp:b1ofl4m) ([raw file](https://github.com/oscarlab/graphene/blob/16ed25262e4424209dc4f56578e15f4de421083c/Pal/src/host/Linux-SGX/tools/ra-tls/ra_tls_verify_epid.c#L226)):*
> ```
> see also ttps://
> ```

typo

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)* status: 0 of 1 files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " and "TODO" found in commit messages' one-liners (waiting on @mkow)

___
*[Pal/src/host/Linux-SGX/tools/ra-tls/ra_tls_verify_epid.c, line 226 at r1](https://reviewable.io/reviews/oscarlab/graphene/2575#-MfFbXCj2ZAXvbPd7nho:-MfHBXtN49DbWat3OSL4:b-896fix) ([raw file](https://github.com/oscarlab/graphene/blob/16ed25262e4424209dc4f56578e15f4de421083c/Pal/src/host/Linux-SGX/tools/ra-tls/ra_tls_verify_epid.c#L226)):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> ```
> see also ttps://
> ```

typo
</blockquote></details>

Done.

**Review by mkow — DISMISSED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required), "fixup! " and "TODO" found in commit messages' one-liners

**Review by mkow — DISMISSED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)* status: all files reviewed, all discussions resolved, "TODO" found in commit messages' one-liners

**Review by mkow — APPROVED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene/2575)* status: :shipit: complete! all files reviewed, all discussions resolved
